src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * ExitCommand.java
+ * This class represents a command to exit the Duke application.
+ * It allows the user to terminate the application gracefully.
+ */
+
+package duke.command;
+
+import duke.Ui;
+import duke.task.Storage;
+import duke.task.TaskList;
+
+public class ExitCommand extends Command {
+    @Override
+    public String execute(TaskList tasks, Ui ui, Storage storage) {
+        return ui.showGoodbyeMessage();
+    }
+
+    @Override
+    public boolean isExit() {
+        return true;
+    }
+}
```

Comment:
I like how you split the different commands into classes for better OOP. Good job!